src/components/common/InsightCard.tsx

```diff
@@ -10,10 +12,34 @@ interface Props {
 
 const InsightCard = ({ favicon, insightData }: Props) => {
   const router = useRouter();
+  const { recommendPostResponse, setRecommendPostResponse } =
+    useCalendarPostResponseStore();
+
+  const handleClick = () => {
+    const updatedInsightList = recommendPostResponse.remindInsightList.map(
+      (insight: Insight) =>
+        insight.insightId === insightData.insightId
+          ? {
+              ...insight,
+              isRead: true,
+            }
+          : insight,
+    );
+    const dataWithFlag: CalendarPostResponse = {
+      ...recommendPostResponse,
+      remindRead: updatedInsightList.filter(
+        (insight) => insight.isRead === false,
+      ).length,
+      remindInsightList: updatedInsightList,
+    };
+    setRecommendPostResponse(dataWithFlag);
+    router.push(`/insight/${insightData.insightId}`);
+  };
+
   return (
     <Wrapper
-      opacity={insightData.todayRead ? 0.6 : 1}
-      onClick={() => router.push(`/insight/${insightData.insightId}`)}
+      opacity={insightData.isRead === true ? 0.6 : 1}
```

Comment:
```suggestion
      opacity={insightData.isRead ? 0.6 : 1}
```
`InsightCard.tsx` 파일이랑 `SummaryInsightCard.tsx` 파일 이 코드 부분만 수정한 뒤에 다른 부분은 백엔드 API 업데이트 후 수정하면 좋을 것 같습니다!

Reply:
`opacity={insightData.isRead ? 0.6 : 1}` 로 반영 완료했습니다!